### ISSUE TITLE
Npm error (conflict baucis/mongoose)

### ISSUE THREAD

**Issue by FedeG** [Icinetic/baucis-swagger2#6]
The error is:

```
npm ERR! Linux 3.16.0-4-amd64
npm ERR! argv "node" "/usr/bin/npm" "install"
npm ERR! node v0.10.38
npm ERR! npm  v2.11.2
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package mongoose does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer baucis@1.5.0 wants mongoose@3 - 4.0.4
```

This error is caused because moongose already have version 4.0.6 stable but baucis allows moongose ​​from version 3 to 4.0.4.

**Comment by pjmolina** [comment]
Thanks Fede. The error is already reported to baucis at https://github.com/wprl/baucis/issues/266
Closing this here: nothing to fix on baucis-swagger2.

**Comment by pjmolina** [comment]
Hi @FedeG The Issue has been fixed at `baucis@1.5.1`
